packages/inetutils.rb

```diff
@@ -40,5 +42,27 @@ def self.build
 
   def self.install
     system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
+    system "patchelf --set-rpath #{CREW_LIB_PREFIX} #{CREW_DEST_PREFIX}/bin/ping"
+    system "patchelf --set-rpath #{CREW_LIB_PREFIX} #{CREW_DEST_PREFIX}/bin/ping6"
+    system "patchelf --set-rpath #{CREW_LIB_PREFIX} #{CREW_DEST_PREFIX}/bin/traceroute"
+  end
+
+  def self.preinstall
+    system "sudo chown chronos #{CREW_PREFIX}/bin/ping" \
+      if File.exist? "#{CREW_PREFIX}/bin/ping"
+    system "sudo chown chronos #{CREW_PREFIX}/bin/ping6" \
+      if File.exist? "#{CREW_PREFIX}/bin/ping6"
+    system "sudo chown chronos #{CREW_PREFIX}/bin/traceroute" \
+      if File.exist? "#{CREW_PREFIX}/bin/traceroute"
+  end
+
+  def self.postinstall
+    puts 'Settings permissions for utilities using sudo'.orange
+    system "sudo chown root #{CREW_PREFIX}/bin/ping"
+    system "sudo chown root #{CREW_PREFIX}/bin/ping6"
+    system "sudo chown root #{CREW_PREFIX}/bin/traceroute"
+    system "sudo chmod 4755 #{CREW_PREFIX}/bin/ping"
+    system "sudo chmod 4755 #{CREW_PREFIX}/bin/ping6"
+    system "sudo chmod 4755 #{CREW_PREFIX}/bin/traceroute"
```

Comment:
Shouldn't need to change ownership to root.

Reply:
Also, you shouldn't need to use sudo to change permissions if the ownership is already chronos.

Reply:
hmmm. inetutils' ping won't work otherwise:
```
chronos@localhost /usr/local/lib/crew/packages $ /bin/ping 8.8.8.8
PING 8.8.8.8 (8.8.8.8) 56(84) bytes of data.
64 bytes from 8.8.8.8: icmp_seq=1 ttl=117 time=13.2 ms
64 bytes from 8.8.8.8: icmp_seq=2 ttl=117 time=15.8 ms
^C
--- 8.8.8.8 ping statistics ---
2 packets transmitted, 2 received, 0% packet loss, time 1001ms
rtt min/avg/max/mdev = 13.253/14.571/15.890/1.324 ms
chronos@localhost /usr/local/lib/crew/packages $ /usr/local/bin/ping 8.8.8.8
ping: Lacking privilege for icmp socket.
```

Reply:
The only alternative I see is to use setcap, which requires sudo too.

e.g. ```sudo /usr/local/sbin/setcap  cap_net_raw+ep /usr/local/bin/ping```

This is what google's default ping has done to it:
```
chronos@localhost /usr/local/lib/crew/packages $ /usr/local/sbin/getcap /bin/ping
/bin/ping = cap_net_raw+ep
```

Reply:
(This would be the cleaner solution, not requiring any use of ```chown```.

Reply:
Alternately one could implement a wrapper script which uses sudo, just like is used by the gparted package, but that is its own hack...

Reply:
e.g.

```
#!/bin/bash
sudo -E capsh --caps="cap_net_raw+eip cap_setpcap,cap_setuid,cap_setgid+ep" \
    --keep=1 --user=nobody --addamb=cap_net_raw -- \
    -c "/usr/local/bin/ping $@"
```

Reply:
This is the direction I'm going... but I'm getting capsh errors...

Reply:
wrapper scripts appear to work with newer libcap